Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -60,6 +60,8 @@
  */
 class Scripts
 {
+    protected static ?string $buildPhpCommand = null;
```

Comment:
```suggestion
    protected static ?string $builtPhpCommand = null;
```
as it _has been built_ if it's in there. 😎

Reply:
yes mister english teacher ;) ^^